Functions/BIDS/xASL_bids_BIDSifyM0.m

```diff
@@ -78,6 +78,37 @@
 
 jsonOut.RepetitionTimePreparation = jsonOut.RepetitionTime;
 
+% Look-Locker acquisition for Philips has often TR given as the length of a single readout, not as the entire cycle
```

Comment:
Is this the exact same addition as above? Then please move this to a separate function `xASL_BIDS_BIDSifyLookLocker`